### PR TITLE
Unified multiple Docker Compose files into one for easier usage

### PR DESCRIPTION
**Pull request by croumegous**
Remove all the different docker compose files in scripts folder, to use a single docker compose (in root directory).
Also moved documentation to run the end to end demo in the main readme to better onboard new users.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+version: "3.7"
+
```
Comment (fozziethebeat) [inline docker-compose.yaml:2]: I assume `docker compose up` would bring up the full stack yes?

**Review by fozziethebeat — APPROVED** [review]
I agree that this is quite a bit cleaner and resolves a lot of cross compose referencing.  As long as my question/comment does what I think it does, I like this.

You'll still need review from Yannic or Andreas to finish.

File: docker-compose.yaml (hunk 1 above)
Comment (croumegous) [reply docker-compose.yaml:2]: Yes, exactly

**Review by yk — APPROVED** [review]
awesome, thank you very much!

**Pull request merged**